libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/gpio_irq_api.c

```diff
@@ -0,0 +1,148 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stddef.h>
+#include "cmsis.h"
+#include "gpio_irq_api.h"
+#include "pinmap.h"
+#include "mbed_error.h"
+
+#define EDGE_NONE (0)
+#define EDGE_RISE (1)
+#define EDGE_FALL (2)
+#define EDGE_BOTH (3)
+
+static gpio_irq_handler irq_handler;
+
+
+typedef struct gpio_channel {
+    uint32_t pin_mask;      // bitmask representing which pins are configured for receiving interrupts
+    uint32_t channel_ids;   // mbed "gpio_irq_t gpio_irq" field of instance
+    uint32_t channel_pin;   // pin number in port group
+    gpio_irq_t * obj;
+} gpio_channel_t;
+
+static gpio_channel_t channel;
+
+#ifdef __cplusplus
+extern "C"{
+#endif
+void EXTI_Handler()
+{
+    // To Do
+    //irq_handler(channel.channel_ids);
```

Comment:
Yes, but now it is not implemented. And we will implement this function next version.